scss/_utilities_vertical-spacing.scss

```diff
@@ -17,4 +17,42 @@
       }
     }
   }
+
+  .u-sv--shallow,
+  .u-sv--regular,
+  .u-sv--deep {
+    &::after {
+      @extend %u-vertical-spacer;
+    }
+  }
+
+  .u-sv--shallow {
+    &::after {
+      margin-top: calc($spv--strip-shallow / 2 - 1px); // subtracting the element height, which is set to 1px above.
+
+      @media (min-width: $breakpoint-large) {
+        margin-top: calc($spv--strip-shallow - 1px); // subtracting the element height, which is set to 1px above.
+      }
+    }
+  }
+
+  .u-sv--regular {
+    &::after {
```

Comment:
These sections are quite repetitive (I guess the only change is the variable used), maybe it would be possible to extract it to some mixin and pass variable as a parameter, to avoid copying these sections of the code?

Reply:
I don't think it's possible, because we use exact SCSS vars in our code. I can't run an eval command to figure out a variable name by adding another variable into it.

Reply:
I was thinking about something like:

```
@mixin vf-section-spacing-vertical($spacing) {
    &::after {
      margin-top: calc($spacing / 2 - 1px);

      @media (min-width: $breakpoint-large) {
        margin-top: calc($spacing - 1px);
      }
    }
}

.u-sv--regular {
   @include vf-section-spacing-vertical($spv-strip--regular);
}
```

I think something like that should work - you don't have to evaluate variable name, just pass it as a parameter to mixin.

Reply:
Should it be a part of `vf-u-vertical-spacing` or create a new file for that mixin so people would have to include it separately?

Reply:
For now we only need it here I think, so just make it part of this file. We can always extract it if we start needing it elsewhere.

Reply:
As it is only needed in this util for now let's keep it here. It can always be extracted to some common file if we need to reuse it elsewhere.